src/domain/community/virtualContributor/vcProfilePage/model.ts

```diff
@@ -0,0 +1,27 @@
+import { BasicSpaceProps } from '../components/BasicSpaceCard';
+
+export type VCProfilePageViewProps = {
```

Comment:
I agree on extracting the types/interfaces in separate files. 
However, what would be the convention? What fi we have a couple of them like ProfileProps but also ProfileFormProps?
This `model` could be misleading and usually we're not putting every component in a separate folder.
It's not critical for this PR but let's decide on how we're going to approach it in the future together with @ccanos.

Reply:
The motivation behind the name `model.ts` is coming due the fact that this file lives in the `vcProfilePage` folder which means this `model.ts` is containing types for every file that lives in that `vcProfilePage` folder. The `model.ts` file can include many types as long as they are strictly related to the files in the `vcProfilePage` folder.